frontend/setupChelonia.js

```diff
@@ -176,7 +176,8 @@ const setupChelonia = async (): Promise<*> => {
           'chelonia/contract/hasKeysToPerformOperation',
           'gi.actions/identity/kv/initChatRoomUnreadMessages', 'gi.actions/identity/kv/deleteChatRoomUnreadMessages',
           'gi.actions/identity/kv/setChatRoomReadUntil',
-          'gi.actions/identity/kv/addChatRoomUnreadMessage', 'gi.actions/identity/kv/removeChatRoomUnreadMessage'
+          'gi.actions/identity/kv/addChatRoomUnreadMessage', 'gi.actions/identity/kv/removeChatRoomUnreadMessage',
+          'gi.actions/identity/kv/markAsUnread'
```

Comment:
What was the reason this selector was added to `allowedSelectors`?

`allowedSelectors` is only for selectors inside the app that can be called from contracts. But AFAICT `'gi.actions/identity/kv/markAsUnread'` is only ever called from `ChatMain.vue`, not from a contract.